### ISSUE TITLE
In Iframed installations provide a button to open in new tab

### ISSUE THREAD

**Issue by DanielHabenicht** [lovasoa/whitebophir#206]
In an Iframed installation of WBO there is no easy way to open the whiteboard in an extra tab to make room for working with whatever else there is in the embedding application. 

It would be nice to have a button that is only show when WBO is in an Iframe, that allows to open the same board in a new tab.

**Comment by lovasoa** [comment]
This is probably more something that should be handled by the embedding application, that created the iframe, wouldn't it ?
WBO should not have to worry about whether it is embedded or not.

**Comment by DanielHabenicht** [comment]
I agree somewhat, but in Jitsi its quite cumbersome to add a new button :D
If I make it configurable?

**Comment by lovasoa** [comment]
Maybe a compromise would be a "share" tool, that allows to copy the URL of the current board, together with a small message like `to collaborate on a board with someone, share this URL with them`. This would be a useful feature even when not embedded, but when embedded, this would allow copying the URL and pasting it in a new tab. What do you think ?

**Comment by lovasoa** [comment]
This is the way google docs does it: 
![image](https://user-images.githubusercontent.com/552629/121520626-6abded80-c9f3-11eb-9a38-ab594afb4d94.png)
